### PR TITLE
doc: add ci build status of master

### PR DESCRIPTION
**Pull request by nashif**
Signed-off-by: Anas Nashif <anas.nashif@intel.com>

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -5,6 +5,9 @@ Zephyr Project
 
    <a href="https://bestpractices.coreinfrastructure.org/projects/74"><img
    src="https://bestpractices.coreinfrastructure.org/projects/74/badge"></a>
+   <img
```
Comment (dbkinder) [inline README.rst]: Is there a link you can add (as there is for the bestpractices badge) that folks can click on for more details?

Comment (dbkinder) [reply README.rst]: maybe this?  https://app.shippable.com/github/zephyrproject-rtos/zephyr/dashboard

**Review by dbkinder — CHANGES_REQUESTED** [review]
add a link to the badge to see the shippable dashboard?

File: README.rst (hunk 1 above)
Comment (nashif) [reply README.rst]: this will be just confusing, I am planning to have something more specific for this, but would start with the badge without a link.

**Review by dbkinder — APPROVED** [review]
Later then. Most badges I've seen have a link for details.

**Pull request merged**